src/main/java/org/eclipse/tractusx/ssi/lib/did/resolver/DidResolverException.java

```diff
@@ -21,13 +21,10 @@
 
 package org.eclipse.tractusx.ssi.lib.did.resolver;
 
-/** The type Did resolver exception. */
-public class DidResolverException extends Exception {
-
-  private static final long serialVersionUID = 1L;
+import org.eclipse.tractusx.ssi.lib.exception.SSIException;
 
-  /** Instantiates a new Did resolver exception. */
-  public DidResolverException() {}
+/** The type Did resolver exception. */
+public class DidResolverException extends SSIException {
```

Comment:
Class should be moved, e.g. to package **org.eclipse.tractusx.ssi.lib.exception.did**